tests/pools/cluster/dynamic.test.js

```diff
@@ -5,7 +5,7 @@ const max = 3
 const pool = new DynamicClusterPool(
   min,
   max,
-  './tests/worker/cluster/testWorker.js',
+  './tests/worker-files/cluster/testWorker.js',
```

Comment:
It's more the worker scripts here.

Reply:
I also thought about the name `fixtures`
I think this is a common name for files for tests that are used within them, but not the tests directly